src/v/raft/tests/raft_fixture.h

```diff
@@ -272,13 +272,18 @@ class raft_node_instance : public ss::weakly_referencable<raft_node_instance> {
 
     service_t& get_service() { return _service; }
 
+    void set_default_recovery_read_size(size_t bytes) {
+        _default_recovery_read_size.update(std::move(bytes));
+    }
+
 private:
     model::node_id _id;
     model::revision_id _revision;
     prefix_logger _logger;
     ss::sstring _base_directory;
     config::mock_property<size_t> _max_inflight_requests{16};
     config::mock_property<size_t> _max_queued_bytes{1_MiB};
+    config::mock_property<size_t> _default_recovery_read_size{32_KiB};
```

Comment:
any reason we change it for existing tests?

Reply:
no particular reason, i will make sure it is the same as before